opsdroid/web.py

```diff
@@ -13,8 +13,21 @@
 
 
 class Web:
-    """Create class for opsdroid Web server."""
+    """A Web class
```

Comment:
Could you make this more descriptive? This line is often used as a short version of the docstring.